Anamnesis/Actor/Posing/Visuals/SkeletonVisual3d.cs

```diff
@@ -642,6 +634,10 @@ public async Task SetActor(ActorMemory actor)
 					break;
 				}
 			}
+
+			// Notify that the skeleton has changed.
+			// All properties that depend on the skeleton are prompted to update.
+			this.RaisePropertyChanged(string.Empty);
```

Comment:
Does String.Empty just work here? Im gonna be honest I havent interacted with the Property system much in Ana lol.

Reply:
"The [PropertyChanged](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.inotifypropertychanged.propertychanged?view=net-6.0#system-componentmodel-inotifypropertychanged-propertychanged) event can indicate all properties on the object have changed by using either null or [String.Empty](https://learn.microsoft.com/en-us/dotnet/api/system.string.empty?view=net-6.0#system-string-empty) as the property name in the [PropertyChangedEventArgs](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.propertychangedeventargs?view=net-6.0)."

**Source:** https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.inotifypropertychanged.propertychanged?view=net-6.0

Reply:
*Writes that down*
That is, very good to know.